src/redismodule.h

```diff
@@ -707,19 +707,19 @@ static const RedisModuleEvent
 #define _REDISMODULE_SUBEVENT_CRON_LOOP_NEXT 0
 #define _REDISMODULE_SUBEVENT_SWAPDB_NEXT 0
 
-#define REDISMODULE_SUBEVENT_CLUSTER_ASM_IMPORT_STARTED 0
-#define REDISMODULE_SUBEVENT_CLUSTER_ASM_IMPORT_FAILED 1
-#define REDISMODULE_SUBEVENT_CLUSTER_ASM_IMPORT_COMPLETED 2
-#define REDISMODULE_SUBEVENT_CLUSTER_ASM_MIGRATE_STARTED 3
-#define REDISMODULE_SUBEVENT_CLUSTER_ASM_MIGRATE_FAILED 4
-#define REDISMODULE_SUBEVENT_CLUSTER_ASM_MIGRATE_COMPLETED 5
-#define REDISMODULE_SUBEVENT_CLUSTER_ASM_MIGRATE_MODULE_PROPAGATE 6
-#define _REDISMODULE_SUBEVENT_CLUSTER_ASM_NEXT 7
+#define REDISMODULE_SUBEVENT_CLUSTER_SLOT_MIGRATION_IMPORT_STARTED 0
```

Comment:
maybe MIGRATION word is not necessary for this part?
```
REDISMODULE_SUBEVENT_CLUSTER_SLOT_IMPORT_STARTED
REDISMODULE_SUBEVENT_CLUSTER_SLOT_IMPORT_FAILED
REDISMODULE_SUBEVENT_CLUSTER_SLOT_IMPORT_COMPLETED
REDISMODULE_SUBEVENT_CLUSTER_SLOT_MIGRATE_STARTED
REDISMODULE_SUBEVENT_CLUSTER_SLOT_MIGRATE_COMPLETED
```

Reply:
but i am not object to current names

Reply:
I wanted to do it but if we follow the convention, we need to change top event to REDISMODULE_CLUSTER_SLOT, it sounds a bit unclear. Similarly, data structures like "RedisModuleClusterSlotInfo".

Reply:
ok, let's use current names